package.json

```diff
@@ -11,6 +11,7 @@
     "pretest": "npm run lint",
     "test": "karma start",
     "test:watch": "karma start --no-single-run --auto-watch",
+    "version": "json -I -f assets/manifest.json -e \"this.version='`json -f package.json version`'\" && git add assets/manifest.json",
```

Comment:
I think we have to do the `git add` here. I actually forgot it [at first](https://github.com/josephfrazier/browser-extension/commit/3e513acd95c80209bdd488a1ddad0df229507c58), but then the `assets/manifest.json` change wasn't in the generated commit when I tested `npm version patch`. Here's the relevant section of the [documentation](https://docs.npmjs.com/cli/version):

> 4\. Run the `version` script. These scripts have access to the new `version` in package.json (so they can incorporate it into file headers in generated files for example). Again, scripts should explicitly add generated files to the commit using `git add`.